models/notifications.js

```diff
@@ -0,0 +1,39 @@
+const firestore = require('../utils/firestore')
+const notificationModal = firestore.collection('notifications')
+/**
+ *  Get  the notifications data for current loggedIn user
+ *
+ * @param currentPage { Integer }: Page for which notifications are to be fetched
+ * @param limit { Integer }: Number of Notifications per page
+ * @param userId { String }: userId of current loggedIn user
+ * @return {Promise<notificationModal|Array>}}
+ */
+
+module.exports.fetchNotifications = async ({ currentPage, limit, userId }) => {
```

Comment:
Please seperate function definitions from export statements to increase readability and for future changes.